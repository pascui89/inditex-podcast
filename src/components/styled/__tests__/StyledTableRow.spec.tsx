import React from 'react';
import { render } from '@testing-library/react';
import { StyledTableRow } from '../StyledTableRow';
import '@testing-library/jest-dom/extend-expect';

global.React = React;

describe('StyledTableRow', () => {

  it('should render', () => {
    const { container } = render(<StyledTableRow/>);
    expect(container).toMatchSnapshot();
  });
});
