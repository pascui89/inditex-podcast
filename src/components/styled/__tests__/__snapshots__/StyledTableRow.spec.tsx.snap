// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledTableRow should render 1`] = `
<div>
  <tr
    class="MuiTableRow-root css-1enotpu-MuiTableRow-root"
  />
</div>
`;
